### DISCUSSION TITLE
Read binary initial heads fails in multi layer disv model.

### DISCUSSION THREAD

**Discussion by RyanConway91** [MODFLOW-ORG/modflow6#1694] [General]
**Describe the bug**

Read binary initial heads fails in multi layer disv model. Binary read worked fine when model was 1 layer. Initial heads read from external float array works fine. 

The intial heads file I am trying to read is the written by the model (so dims are right)

Model is 1 kstp, 1 kper

Here is .ic6:

# MODFLOW6 Initial Conditions
BEGIN GRIDDATA
  STRT
  OPEN/CLOSE starting.hds (BINARY) IPRN  -1  Starting Heads
END GRIDDATA

MODFLOW version = 6.4.3 02/07/2024

**Comment by christianlangevin** [comment]
Sadly, this is a known issue.  The following description is included in the input/output guide:

A consequence of the way binary input files have been implemented in MODFLOW 6, simulated dependent variable binary output (for example, head and concentration) cannot be used as binary array input for a model. Instead, simulated dependent variable binary output must be processed and split into separate binary files for each layer or combined into a single array equal to the size of the grid (for DIS grids this would be an array equal to NCOL * NROW * NLAY).

**Reply by RyanConway91** [reply]
Huh I swear this used to work. Is this a new change?
